kafka-testing/src/test/resources/kafka_servers/kafka_test_server_polling.properties

```diff
@@ -0,0 +1,36 @@
+# =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+# kafka bootstrap servers comma separated
+# e.g. localhost:9092,host2:9093
+# =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+kafka.bootstrap.servers=localhost:9092
+
+kafka.producer.properties=kafka_servers/kafka_producer.properties
+kafka.consumer.properties=kafka_servers/kafka_consumer.properties
+
+# --------------------------------------------------------------------
+# Optional local consumer properties common/central to all test cases.
+# These can be overwritten by the tests locally.
+# --------------------------------------------------------------------
+# If this property is set, then the consumer does a commitSync after reading the message(s)
+# Make sure you don't set both commitSync and commitAsync to true
+consumer.commitSync = true
+# If this property is set, then the consumer does a commitAsync after reading the message(s)
+# Make sure you don't set both commitSync and commitAsync to true
+consumer.commitAsync = false
+# All records those were read are dumped to this specified file path
+# This path can be a relative path or an absolute path. If the file
+# does not exist, it creates the file and dumps the records
+consumer.fileDumpTo= target/temp/demo.txt
+# If this property is set to true, all records are shown in the response.
+# When dealing with large number of records, you might not be interested
+# in the individual records, but interested in the recordCount
+# i.e. total number of records consumed
+consumer.showRecordsConsumed=false
+# That means if any record(s) are read, then this counter is reset to 0(zero) and the consumer
+# polls again. So if no records are fetched for a specific poll interval, then the consumer
+# gives a retry retrying until this max number polls/reties reached.
+consumer.maxNoOfRetryPollsOrTimeouts = 5
+
```

Comment:
@dandalavinod , can you add an entry with `consumer.pollingTime` in this config file, which will override the default value of 500 and the test should pass.
e.g.
> consumer.pollingTime = 1000

Reply:
@authorjapps do I miss understand the current implementation, I thought the default now will be taken from KafkaConstants and will be 100ms?

Reply:
@ddingel i notice that effectiveLocalConfigs.getPollingTime() is returning NULL when its not set in properties. Sorry i should have tested this out early on without assuming. Hence you could see i explicitly set the default to 500L and believe would not require the number of runs increase you suggested.
Long durationForPolling = ofNullable(effectiveLocalConfigs.getPollingTime()).orElse(500L);

@authorjapps the properties file you referred is explicitly testing scenario when the property for pollingTime is not set; and i notice there is KafkaConsumeTest.java which refers to kafka_test_server.properties which has pollingTime set to 3000ms to cover the scenario where its explicitly set.

Reply:
> the properties file you referred is explicitly testing scenario when the property for pollingTime is not set;

yep. 

>  which refers to kafka_test_server.properties which has pollingTime set to 3000ms 

understood.

Reply:
Hi @ddingel,
>@authorjapps do I miss understand the current implementation, I thought the default now will be taken from KafkaConstants and will be 100ms?

if e.g. `consumer.pollingTime = 1000` is provided, then it will use this(1000), otherise it defaults to `500 mili sec` which a constant hardcoded.

This change also means:
`consumer.pollingTime = 1000 or 100` if provided, will be used for both purpose.
1) initialPollWaitingForConsumerGroupJoin = 1000 or 100 milisec, retries 10 times
2) Subsequent pollingTime = 1000 or 100 milisec(configurable as usual)

Without this changes/earlier implementation:
1) initialPollWaitingForConsumerGroupJoin = 500 milisec HARD CODED, retries 10 times
   1.1)This is blocking other users like @dandalavinod as he needs slightly more time to rejoin e.g. 1000milisec
2) Subsequent pollingTime = configurable(100, 500, 1000, 5000 etc...)

Is this going to really affect you? If this is really a breaking change, we need to separate these two configs. But can you check n verify please?